### ISSUE TITLE
drop series problem

### ISSUE THREAD

**Issue by wiku-PSNC** [influxdata/influxdb#9246]
It seems "drop series" command takes no effect; I would expect after "drop series ..." the command "show series ... " with the same conditions will not return any records.

### Bug report
OS: Debian GNU/Linux 9
```
Connected to http://localhost:8086 version 1.4.2
InfluxDB shell version: 1.4.2
> use adva_temp
Using database adva_temp
> drop series where "tsnmp_agent" = 'dwdm3-bernbern'
> show series where "tsnmp_agent" = 'dwdm3-bernbern'
key
---
temperature,tobject=0,toid_suffix=1.0.0.0.4,tshelf=1,tshelfslot=shelf\ \ 1\ slot\ \ 0,tslot=0,tsnmp_agent=dwdm3-bernbern
temperature,tobject=0,toid_suffix=1.1.0.0.6,tshelf=1,tshelfslot=shelf\ \ 1\ slot\ \ 1,tslot=1,tsnmp_agent=dwdm3-bernbern
temperature,tobject=0,toid_suffix=1.17.0.0.6,tshelf=1,tshelfslot=shelf\ \ 1\ slot\ 17,tslot=17,tsnmp_agent=dwdm3-bernbern
temperature,tobject=0,toid_suffix=1.19.0.0.6,tshelf=1,tshelfslot=shelf\ \ 1\ slot\ 19,tslot=19,tsnmp_agent=dwdm3-bernbern
temperature,tobject=0,toid_suffix=1.3.0.0.6,tshelf=1,tshelfslot=shelf\ \ 1\ slot\ \ 3,tslot=3,tsnmp_agent=dwdm3-bernbern
temperature,tobject=0,toid_suffix=1.33.0.0.6,tshelf=1,tshelfslot=shelf\ \ 1\ slot\ 33,tslot=33,tsnmp_agent=dwdm3-bernbern
temperature,tobject=0,toid_suffix=1.34.0.0.6,tshelf=1,tshelfslot=shelf\ \ 1\ slot\ 34,tslot=34,tsnmp_agent=dwdm3-bernbern
temperature,tobject=0,toid_suffix=1.4.0.0.6,tshelf=1,tshelfslot=shelf\ \ 1\ slot\ \ 4,tslot=4,tsnmp_agent=dwdm3-bernbern
temperature,tobject=16896,toid_suffix=1.3.16896.0.9,tshelf=1,tshelfslot=shelf\ \ 1\ slot\ \ 3,tslot=3,tsnmp_agent=dwdm3-bernbern
temperature,tobject=17152,toid_suffix=1.3.17152.0.9,tshelf=1,tshelfslot=shelf\ \ 1\ slot\ \ 3,tslot=3,tsnmp_agent=dwdm3-bernbern
```

**Comment by jwilder** [comment]
There are some delete fixes coming in 1.4.3.  This looks like it may be a dup of #9161

**Comment by wiku-PSNC** [comment]
version 1.5.0 solves this problem :)
